ajax/libs/loadCSS/package.json

```diff
@@ -0,0 +1,35 @@
+{
+  "name": "loadCSS",
+  "description": "A function for loading CSS asynchronously",
```

Comment:
As you added v1.1.0 assets, it's better to add its version number here next time.